### PR TITLE
Feature/identity service

### PR DESCRIPTION
**Pull request by RomanShidlovsky**
Identity Service using N-layer architecture: DataAccess, BusinessLogic, API layers

Techologies:
- UserManager
- RoleManager
- IdentityServer4

### CODE REVIEW

File: src/Shared/Extensions/SwaggerConfigurationExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+namespace Shared.Extensions;
+
+public static class SwaggerConfigurationExtensions
+{
+    public static void ConfigureSwagger(this IServiceCollection services, IConfiguration configuration)
+    {
+        services.AddSwaggerGen(options =>
+        {
+
```
Comment (Elizaveta564) [inline src/Shared/Extensions/SwaggerConfigurationExtensions.cs]: extra line

File: src/Services/Book/Book.Infrastructure/ServiceExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Book.Infrastructure;
+
+public static class ServiceExtensions
+{
+    public static void ConfigureInfrastructure(this IServiceCollection services, IConfiguration configuration)
+    {
+        
+    }
```
Comment (Elizaveta564) [inline src/Services/Book/Book.Infrastructure/ServiceExtensions.cs:11]: why is it empty

File: src/Services/Book/Book.Application/ServiceExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Book.Application;
+
+public static class ServiceExtensions
+{
+    public static void ConfigureApplication(this IServiceCollection services)
+    {
+        
+    }
```
Comment (Elizaveta564) [inline src/Services/Book/Book.Application/ServiceExtensions.cs:10]: why is it empty

**Review by Elizaveta564 — CHANGES_REQUESTED** [review]

File: src/Services/Book/Book.Infrastructure/ServiceExtensions.cs (hunk 2 above)
Comment (RomanShidlovsky) [reply src/Services/Book/Book.Infrastructure/ServiceExtensions.cs:11]: I implemented identity service before book service, and created these methods along with the application skeleton

**Review by Elizaveta564 — APPROVED** [review]

**Pull request merged**